3 - PB/Documentazione esterna/Specifica Tecnica/Specifica Tecnica.typ

```diff
@@ -73,6 +73,84 @@ Il prodotto offre le seguenti funzionalità principali:
 
 == Database
 
+In questa sezione viene presentato lo schema della base di dati realizzata con PostgreSQL.
+
+Esso è cosi composto:
+
+#figure(
+  image("ER.png", width: 100%),
+  caption: [
+    Schema ER del Database.
+  ],
+)
+
+
+=== Entità
+
+Il database è composto da 6 entità:
+
+- *Product*: rappresenta un prodotto presente all'interno del magazzino. Composto da:
+
+    - ID: identificativo univoco e seriale di un prodotto;
+    - Name: nome del prodotto;
+    - Weight: peso del prodotto;
+    - Width: larghezza del prodotto;
+    - Length: lunghezza del prodotto;
+    - Height: altezza del prodotto.
+
+- *Category*: rappresenta la categoria di appartenenza dei prodotti. Composto da:
+
+    - ID: identificativo univoco e seriale di una categoria;
+    - Name: nome della categoria.
+
+- *Bin*: rappresenta uno spazio del magazzino in cui è possibile inserire un prodotto. Composto da:
+
+    - ID: identificativo univoco e seriale di un bin.
+
+- *Level*: rappresenta un ripiano dello scaffale. Composto da:
+
+    - ID: identificativo univoco e seriale di un ripiano;
+    - Height: altezza del ripiano;
+    - Level_order: ordine del ripiano (rispetto agli altri nello scaffale).
```

Comment:
Level_order: valore che rappresenta la posizione dello scaffale all'interno di uno scaffale, il suo valore è incrementale da un iniziale valore 1 (che rappresenta il primo scaffale del basso) all'ultimo in alto con valore n [questa cosa verificala con gli altri che non sono sicuro], se il suo valore è 0 allora rappresenta una zona a tezza